task/distributed_queue.py

```diff
@@ -72,13 +79,16 @@ def __init__(self, amqp_server, queues, **kwargs):
         if no_amqp:
             raise ImportError('pika is not available')
 
+        if secrets_dir is None:
+            secrets_dir = DEFAULT_SECRETS_DIR
```

Comment:
Why not just use `DEFAULT_SECRETS_DIR` as default value in `__init__` definition?

Reply:
You are right, this is nicer. Fixed.